### PR TITLE
Fix #3629 - svdb_origin tags

### LINKED ISSUE #3629: svdb_origin
SVDB uses a VCF tag `svdb_origin` that appears synonymous in function to `FOUND_IN` - a pipe separated list of caller tags.
Parsing this is useful in particular for variants found by all of multiple callers, with set=Intersection.
Screenshot 2022-10-11 at 17 12 03

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality.

`svdb_origin` is now parsed in addition to `FOUND_IN` and `set`.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. pick a case merged with a recent svdb version, producing svdb_origin tags, and a variant found by all callers
2. note it is shown as called by some callers not used in the analysis
3. reload with this patch, and see only the actual callers found

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN

### CODE REVIEW

**Comment by dnil** [comment]
Should be ready for review, given that the tests pass naturally. Fixes the issue on 4.60 testing (really it was there before summer as well, when callers were still parsed) with set=Intersection adding in callers known to Scout but not really part of the pipeline.

File: scout/parse/variant/callers.py [hunk 1]
```diff
@@ -8,6 +8,11 @@
 def parse_callers(variant, category="snv"):
     """Parse how the different variant callers have performed
 
+    Caller information can be passed in one of three ways, in order of priority:
```
Comment (northwestwitch) [inline scout/parse/variant/callers.py:11]: Very good with the explanation, it's a complicated matter

**Review by northwestwitch — APPROVED** [review]
👍🏻 Looks good!

Would be nice to add something about svdb also in the docs, perhaps this one: https://github.com/Clinical-Genomics/scout/blob/main/docs/user-guide/variants.md

Doesn't have to be in this PR though

**Comment by dnil** [comment]
Looks good so far. Here is justhusky loaded on stage with an "Intersection" variant from all three callers:
Screenshot 2022-10-12 at 09 30 14

**Comment by dnil** [comment]
> Would be nice to add something about svdb also in the docs, perhaps this one: https://github.com/Clinical-Genomics/scout/blob/main/docs/user-guide/variants.md

I reformatted that part about SVDB a little; it also has a link to the repo. I'm all for figuring out a good way to connect tests and documentation here; should be possible to use some auto-doc function to get more of the docstrings from tests out. Then we could get going documenting VCF info that Scout can actually display, and the content expected. But another PR.

**Pull request merged**